vars/perfSearchBestConfigParallelPipeline.groovy

```diff
@@ -0,0 +1,354 @@
+                                                    def test_config = groovy.json.JsonOutput.toJson(pipelineParams.test_config)
+                                                    timeout(time: testRunTimeout, unit: 'MINUTES') { dir('scylla-cluster-tests') {
+
+                                                        sh """
+                                                        #!/bin/bash
+                                                        set -xe
+                                                        env
+
+                                                        rm -fv ./latest
+
+                                                        export SCT_CLUSTER_BACKEND=${params.backend}
+                                                        if [[ -n "${params.region}" ]]; then
+                                                            export SCT_REGION_NAME=${params.region}
+                                                        else
+                                                            export SCT_REGION_NAME=${pipelineParams.region}
+                                                        fi
+                                                        export SCT_CONFIG_FILES=${test_config}
+
+
+                                                        if [[ ! -z "${params.instance_type_db}" ]] ; then
+                                                            export SCT_INSTANCE_TYPE_DB=${params.instance_type_db}
+                                                        fi
+                                                        if [[ ! -z "${params.instance_type_loader}" ]] ; then
+                                                            export SCT_INSTANCE_TYPE_LOADER=${params.instance_type_loader}
+                                                        fi
+                                                        if [[ ! -z "${params.n_loaders}" ]] ; then
+                                                            export SCT_N_LOADERS=${params.n_loaders}
+                                                        fi
+                                                        if [[ ! -z "${params.num_loaders_step}" ]] ; then
+                                                            export SCT_NUM_LOADERS_STEP=${params.num_loaders_step}
+                                                        fi
+                                                        if [[ ! -z "${params.stress_threads_start_num}" ]] ; then
+                                                            export SCT_STRESS_THREADS_START_NUM=${params.stress_threads_start_num}
+                                                        fi
+                                                        if [[ ! -z "${params.add_num_threads_step}" ]] ; then
+                                                            export SCT_ADD_NUM_THREADS_STEP=${params.add_num_threads_step}
```

Comment:
This is not the correct name of the parameter (you've probably renamed it).
According to sct_config it's now: SCT_NUM_THREADS_STEP